Source/LibationWinForms/GridView/GridEntryBindingList.cs

```diff
@@ -24,7 +24,11 @@ namespace LibationWinForms.GridView
 	internal class GridEntryBindingList : BindingList<IGridEntry>, IBindingListView
 	{
 		public GridEntryBindingList() : base(new List<IGridEntry>()) { }
-		public GridEntryBindingList(IEnumerable<IGridEntry> enumeration) : base(new List<IGridEntry>(enumeration)) { }
+		public GridEntryBindingList(IEnumerable<IGridEntry> enumeration) : base(new List<IGridEntry>(enumeration))
+		{
+			SearchEngineCommands.SearchEngineUpdated += (_,_) => ApplyFilter(FilterString);
```

Comment:
This isn't functionally wrong, but it feels philosophically wrong. Something will update the object then persist to the database, possibly within the same method. The persistence will say when it's updated and will then update the search engine. It seems like the persistence-update would be the thing to subscribe to. I'm going off of memory though and you've obviously been more boots on the ground lately. Thoughts?

Reply:
I'm not fond of this solution either, but this bug is a problem...

Here's where we detect changes and re-filter.
https://github.com/rmcrackan/Libation/blob/e52772826af681b3805e292b3bea19602975d305/Source/LibationWinForms/GridView/GridEntryBindingList.cs#L218-L227

The problem is that the item changes before the search has been re-indexed. I feel like this has been an ongoing problem.

Anyway, I'll download an old release tomorrow to see if I can find a time when it was working properly and see what we did then. I'm not in the best state to be thinking about this now.

Reply:
Will `BookUserDefinedItemCommitted` accomplish the same?

> I'm not in the best state to be thinking about this now

Take care of you. Nothing about this is urgent.